### ISSUE TITLE
Update documentation

### ISSUE THREAD

**Issue by garritfra** [clio-lang/clio#13]

**Comment by fuzzthink** [comment]
At the minimum a language reference is needed. It's not useable without it.

**Comment by garritfra** [comment]
You are correct. Documentation should be a crucial part in the development process. It's a thing we want to work on improving. Feel free to help us out!

**Comment by pouya-eghbali** [comment]
There are a few changes in the way the language works, I'm working on them, as soon as it's finished I'm going to do a better documentation. I cannot do a language reference right now because the syntax and the features aren't stable now.

**Comment by pouya-eghbali** [comment]
The current documentation generator we use isn't free anymore, but I still find it the nicest. I believe there's still a free version, however I don't know what kind of limitation there is for free members.

**Comment by garritfra** [comment]
Gitbook? If all hell breaks loose, we can use a static site generator like Gatsby

**Comment by garritfra** [comment]
Take a look at [this](https://www.gatsbyjs.org/starters/ericwindmill/gatsby-starter-docs/)

**Comment by RigottiG** [comment]
Interested to translate to more than one language? I can do the Brazilian Portuguese version of doc.
